Assets/Scripts/Data/Pickup Pup/Currency/HomeSlotsData.cs

```diff
@@ -0,0 +1,157 @@
+﻿/*
+ * Author: Grace Barrett-Snyder 
+ * Description: Stores data for the Player's home slots currency.
+ */
+
+[System.Serializable]
+public class HomeSlotsData : CurrencyData
+{
+    HomeSlot[] slots;
+
+    #region Instance Accessors
+
+    public int VacantSlots
+    {
```

Comment:
This is getting nitpicky, but I feel like it's unnecessary for these methods to be O(n). Assuming this class is the only class changing number of slots it manages (as it should be Law of Demeter), I think it would be best to just use an integer as a backing var